src/UserSpaceInstrumentation/InstrumentProcessTest.cpp

```diff
@@ -40,24 +40,19 @@ constexpr int kFunctionId2 = 43;
 orbit_grpc_protos::CaptureOptions BuildCaptureOptions() {
   orbit_grpc_protos::CaptureOptions capture_options;
 
-  constexpr const char* kFunctionName1 = "SomethingToInstrument";
-  AddressRange range = GetFunctionRelativeAddressRangeOrDie(kFunctionName1);
-  orbit_grpc_protos::InstrumentedFunction* my_function =
-      capture_options.add_instrumented_functions();
-  my_function->set_function_id(kFunctionId1);
-  my_function->set_file_offset(range.start);
-  my_function->set_function_size(range.end - range.start);
-  my_function->set_function_name(kFunctionName1);
-  my_function->set_file_path(orbit_base::GetExecutablePath());
-
-  constexpr const char* kFunctionName2 = "ReturnImmediately";
-  range = GetFunctionRelativeAddressRangeOrDie(kFunctionName2);
-  my_function = capture_options.add_instrumented_functions();
-  my_function->set_function_id(kFunctionId2);
-  my_function->set_file_offset(range.start);
-  my_function->set_function_size(range.end - range.start);
-  my_function->set_function_name(kFunctionName2);
-  my_function->set_file_path(orbit_base::GetExecutablePath());
+  const auto add_function = [&capture_options](std::string_view function_name, int function_id) {
```

Comment:
I'd make this a function in the unnamed namespace. But this is just me not liking lambdas. Feel free to ignore.

Reply:
I don't mind changing it. Done.